core-services/ci-secret-bootstrap/_config.yaml

```diff
@@ -2454,6 +2454,9 @@ secret_configs:
     sa.config-updater.arm01.config:
       field: sa.config-updater.arm01.config
       item: config-updater
+    sa.config-updater.build01.config:
```

Comment:
I guess we need to revert all of them, right?

Reply:
Yes. But this one will help us to recover a manually deleted one.
https://redhat-internal.slack.com/archives/C05T50FMW0H/p1695148857822229?thread_ts=1695144662.791559&cid=C05T50FMW0H